libcloudforensics/providers/azure/internal/account.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Represents an Azure account."""
+
+from time import sleep
+from typing import Optional, Dict
+
+# pylint: disable=import-error
```

Comment:
Could you add a comment about why we're ignoring the `import-error` here?

Reply:
Tbh I couldn't find an actual reason; my local linter fails to import these for some reason. Seems to have happened to other people from the quick search I've done, but I didn't find any fix.

Reply:
I assume pylint complains about this, even though the libs are installed in its virtualenv? Could be that the underlying libs don't expose their members correctly. In that case, please add a note that even though the linters complain, the libs can be imported correctly

Reply:
Correct! I added a note.